### DISCUSSION TITLE
Is there a way to fetch all translations via a single API call?

### DISCUSSION THREAD

**Discussion by MaxDonchenko** [crowdin/crowdin-api-client-js#185] [Q&A] [answered]
From [this thread](https://github.com/crowdin/crowdin-api-client-js/issues/111#issuecomment-992882965) I see that I probably need to build translations and then download them. However, only schema (request/response parameters) is described in docs for this method. It's not quite clear how it should work.

The README in [JS API client](https://github.com/crowdin/crowdin-api-client-js) has some examples but it doesn't cover everything. 

I'm curious is there a quick way to fetch all translations from all files combined together into 1 API call? (or several calls, based on pagination params)

For instance, pseudo API call I imagine:
```
GET /api/v2/projects/{projectId}/translations/languages/{languageId}
```

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @goodwin64!

You could use the [exportProjectTranslation](https://crowdin.github.io/crowdin-api-client-js/classes/Translations.html#exportProjectTranslation) [API](https://developer.crowdin.com/api/v2/#operation/api.projects.translations.exports.post). This API method allows downloading all the translations for a single language.

Could you please describe your use case in more detail? Probably, you could also use the [Over-The-Air Content delivery](https://github.com/crowdin/ota-client-js)
